benchmarks/benchmark_latency.py

```diff
@@ -34,6 +35,10 @@ def save_to_pytorch_benchmark_format(
         write_to_json(pt_file, pt_records)
 
 
+@deprecated(
+    "benchmark_latency.py is deprecated and will be removed in a "
+    "future version. Please use 'vllm bench latency' instead.",
+)
```

Comment:
Is it safe to add this to `main()`? I would asume that the CLI calls this function when using `vllm bench latency`?

(same comment on the other deprecations)